management/management_test.go

```diff
@@ -155,7 +155,7 @@ func TestOptionDefaults(t *testing.T) {
 	assert.Equal(t, "20", perPage)
 
 	includeTotals := v.Get("include_totals")
-	assert.Equal(t, "true", includeTotals)
+	assert.Equal(t, "false", includeTotals)
```

Comment:
Totally fine to keep it like this, however there's also the `assert.False `helper func on the assert pkg.